.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] Great work on updating your `.gitignore` file to include the `node_modules` directory! However, I wanted to kindly remind you one of the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you also need to create and add a `test.md` file to your `.gitignore`.  By doing this you will fulfill all the requested requirements of the project.🎯
![image](https://user-images.githubusercontent.com/107952724/234322453-916485a9-6a76-4875-b99e-cc15f7cbf665.png)